packages/eslint-config-nimble-testing/README.md

```diff
@@ -0,0 +1,67 @@
+# `@nimblehq/eslint-config-nimble-testing`
+
+This package provides ESLint testing rules inheriting from @nimblehq/eslint-config-nimble-core.
+
+## Caution
+
+@nimblehq/eslint-config-nimble-core includes `jest` and `cypress` as the primary test suite stack. Consider if the targeting project is not using those libraries.
```

Comment:
Do you mean the `eslint-config-nimble-testing` includes `jest` and `cypress`? 🤔 

```suggestion
@nimblehq/eslint-config-nimble-testing includes `jest` and `cypress` as the primary test suite stack. Consider if the targeting project is not using those libraries.
```

Reply:
Nice catch, updated in [9f9052e](https://github.com/nimblehq/eslint-config-nimble/pull/49/commits/9f9052ec9620a9d0e0372d48cd01a78419280804)